apps/desktop/src/index.html

```
@@ -986,7 +986,10 @@ <h3 class="text-2xs font-bold text-[var(--text-muted)] uppercase tracking-wider
                                         </div>
                                         <div id="fake-client-custom-container" class="hidden">
                                             <label class="block text-2xs text-[var(--text-muted)] uppercase tracking-wider mb-2">Custom User-Agent</label>
-                                            <input type="text" id="fake-client-custom" placeholder="e.g. MyClient/1.0" data-i18n-placeholder="fakeClientCustomPlaceholder" class="form-control form-control-md">
+                                            <div class="input-paste-wrapper">
+                                                <input type="text" id="fake-client-custom" placeholder="e.g. MyClient/1.0" data-i18n-placeholder="fakeClientCustomPlaceholder" class="form-control form-control-md">
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)